utils/vlog/tests/pb_type.xml

```diff
@@ -0,0 +1,34 @@
+<pb_type blif_model=".subckt test_pb" name="test_pb" num_pb="1">
+  <clock equivalent="false" name="clk" num_pins="1"/>
+  <input equivalent="false" name="rst" num_pins="1"/>
+  <input equivalent="false" name="a" num_pins="1"/>
+  <input equivalent="false" name="b" num_pins="1"/>
+  <input equivalent="false" name="ci" num_pins="1"/>
+  <output equivalent="false" name="q" num_pins="1"/>
+  <output equivalent="false" name="co" num_pins="1"/>
+  <mode name="ADD">
+    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="adder/pb_type.xml"/>
+    <interconn>
+      <direct input="test_pb.a" name="test_pb_a_to_adder_a" output="adder.a"/>
```

Comment:
As you can only have a single driver on any net, you can just name these the same as the output value.